rockcraft/oci.py

```diff
@@ -203,6 +183,58 @@ def set_env(self, env: List[Dict[str, str]]) -> None:
         _config_image(image_path, params)
         emit.message(f"Environment set to {env_list}", intermediate=True)
 
+    def set_control_data(self, metadata: Dict[str, Any]) -> None:
+        """Create and populate the ROCK's control data folder.
+
+        :param metadata: content for the ROCK's metadata YAML file
+        :type metadata: Dict[str, Any]
+        """
+        control_data_destination = "/.rock"
```

Comment:
is this mostly to not show the temp path? I would argue that the emit.progress' where this is used would probably go through so fast they would not be read. In any case I would remove this variable and make progress not print out /.rock and instead emit.progress("Writing ROCK metadata") and the closing message is probably a verbose or debug one.

Reply:
I have a few doubts about your comment:

> is this mostly to not show the temp path? 

Which temp path? `/.rock` is the final path inside the ROCK. 
Are you suggesting we don't print it? (just trying to understand the problem)

Reply:
got it. This was leftover code, cause in the past I was using `umoci insert` instead of `umoci raw add-layer`. Fixed in https://github.com/canonical/rockcraft/pull/31/commits/83945867836ac918ddf148de47b63b34d577a8e2